scripts/export_environment.py

```diff
@@ -91,35 +99,41 @@ def __init__(self, parent: Path, component: Path) -> None:
         )
 
 
-def load_env() -> Tuple[Environment, Optional[str]]:
+def load_env(deployment: Optional[str] = None
+             ) -> Tuple[Environment, Optional[str]]:
     """
     Load environment.py and environment.local.py modules from the project
-    root and the current active deployment directory, call their env()
-    function to obtain the environment dictionary and merge the dictionaries.
-    The entries from an environment.local.py take precedence over those from
-    a corresponding environment.py in the same directory. The modules from
-    the deployment directory take precedence over ones in the project root.
+    root and either the specified deployment or the current active deployment
+    directory, call their env() function to obtain the environment dictionary
+    and merge the dictionaries. The entries from an environment.local.py take
+    precedence over those from a corresponding environment.py in the same
+    directory. The modules from the deployment directory take precedence over
+    ones in the project root.
     """
 
     deployments_dir = root_dir / 'deployments'
     active_deployment_dir = deployments_dir / '.active'
-    if active_deployment_dir.exists():
-        if not active_deployment_dir.is_dir() or not active_deployment_dir.is_symlink():
-            raise InvalidActiveDeployment(active_deployment_dir)
-
+    if deployment is not None or active_deployment_dir.exists():
+        if deployment is not None:
```

Comment:
Try to find a way to avoid duplicating the condition, simplifying the control flow.